libdnf/repo/Repo.hpp

```diff
@@ -166,6 +166,35 @@ struct Repo {
     void initHyRepo(HyRepo hrepo);
     int getAge() const;
 
+    /**
+    * @brief Ask for additional repository metadata to download
+    *
+    * given metadata are appended to the default metadata set when repository is downloaded
+    *
+    * @param metadata metadata name (filelists, other, productid...)
+    */
+    void addToMetadata(const std::string &metadata);
```

Comment:
This is a misleading name to me.
How about to addMetadataTypeToDownload() or something similar?

Reply:
Method renamed (and some other identifiers too for clarity's sake)